mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageIdMapperTest.java

```diff
@@ -535,7 +535,7 @@ public void setFlagsShouldUpdateModSeqWhenMessageIsInOneMailbox() throws Excepti
         sut.save(message1);
 
         MessageId messageId = message1.getMessageId();
-        sut.setFlags(messageId, ImmutableList.of(message1.getMailboxId()), new Flags(Flag.ANSWERED), MessageManager.FlagsUpdateMode.REMOVE);
+        sut.setFlags(messageId, ImmutableList.of(message1.getMailboxId()), new Flags(Flag.ANSWERED), FlagsUpdateMode.ADD);
```

Comment:
It's 100% part of this PR

That's why it had it's separated commit: 

```
 MAILBOX-294 Correct MessageIdMapper test

Removing a flag that was not present lead to NOOP.
```

Actually, this test is a **NOOP** update. Old code update it, not new code. To still assert it well, code need to do a real update. New code modseq behaviour on NOOP update is also tested ;-)